packages/openid4vc/src/shared/models/OpenId4VcJwtIssuer.ts

```diff
@@ -3,11 +3,33 @@ interface OpenId4VcJwtIssuerDid {
   didUrl: string
 }
 
+interface OpenId4VcIssuerX5c {
+  method: 'x5c'
+
+  /**
+   *
+   * Array of base64-encoded certificate strings in the DER-format.
+   *
+   * The certificate containing the public key corresponding to the key used to digitally sign the JWS MUST be the first certificate.
+   */
+  chain: Array<string>
+
+  /**
+   * The issuer of the certificate chain.
+   *
+   * If the issuer is a DNS name the x509_san_dns client_id_scheme is used
+   * Otherwise the x509_san_uri client_id_scheme is used.
+   *
+   * Must be provided when creating x509 certificate protected authorization requests.
+   */
+  issuer?: string
```

Comment:
> Must be provided when creating x509 certificate protected authorization requests.

We're already in x5c method here. So should this be required then? In which case is it optional? For the issuance side it is not optional?

Reply:
@auer-martin  :)